pkg/store/forward/forward.go

```diff
@@ -109,7 +109,12 @@ func (s *Store) WriteMetrics(ctx context.Context, p *store.PartitionedMetrics) e
 				Observe(time.Since(begin).Seconds())
 
 			if resp.StatusCode/100 != 2 {
-				return fmt.Errorf("response was not 200 OK, but %s", resp.Status)
+				mean := timeseriesMeanDrift(timeseries, time.Now().Unix()*1000)
```

Comment:
looking at this again, the second param for `timeseriesMeanDrift` is seconds, not milliseconds, so we don't need the `*1000`